zeppelin-web/src/components/noteName-create/visible.directive.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+angular.module('zeppelinWebApp').directive('visible', function () {
+    return {
+        restrict: 'A',
+        link: function(scope, elem, attrs) {
+        	// Add some listeners
+        		$(elem).on('show.bs.modal',function(e) {
+        				var generatedName = generateName();
+        				scope.notename = 'note ' + generatedName;
```

Comment:
Would love to have 'note ' into 'Note ' so that act the same as the backend name generation